website_blog_layout_options/models/blog_category.py

```diff
@@ -0,0 +1,13 @@
+# -*- coding: utf-8 -*-
+# © 2016 Therp BV <http://therp.nl>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openerp import api, fields, models
+
+
+class BlogCategory(models.Model):
+    _name = 'blog.category'
+    _description = ''
+
+    name = fields.Char(String='name', help="Name of this category")
+    post_ids = fields.Many2many('blog.post', string='Posts')
```

Comment:
take care that this uses that same table (and inverted columns) as `blog.post#category_id`, otherwise, you'll get two lists and that's probably not what you want